### ISSUE TITLE
[QUESTION] How to mute or hide the built in terminal?

### ISSUE THREAD

**Issue by chwba** [chriskiehl/Gooey#703]
The reason I want to do this, is that I use loguru with colored logs, which are not displayed correctly.

**Comment by eladeyal-intel** [comment]
Check out the rich terminal! It supports color output.

**Comment by chwba** [comment]
Yeah, I had taken a look at it just now, doesn't seem to work very good with loguru, except for bold

**Comment by chwba** [comment]
Also just noticed, that Gooey is consuming the whole stdout/stderr, so the console remains silent - Either I would like to get Gooeys terminal to work with the coloring of https://github.com/Delgan/loguru or detach Gooey from stdout/stderr, to at least get the colored prints in the terminal - Furthermore - if detaching/closing Gooey until the script finished running, we would be able to use `input()` again.

**Comment by chwba** [comment]
Any news on this one? Its quite a no go for me if I cant get the colored logging loguru provides. Can we pass stdout/stderr back to the console in any way?

**Comment by chwba** [comment]
@chriskiehl any ideas?

**Comment by eladeyal-intel** [comment]
Try setting colorize in the logger configuration

**Comment by chwba** [comment]
> Try setting colorize in the logger configuration

Thats active by default.

**Comment by eladeyal-intel** [comment]
>> Try setting colorize in the logger configuration
> Thats active by default & the output is properly colorized inside the console of course, just not in Gooey.

It is possible that loguru has different defaults in place when running in the console and running inside Gooey. Try setting colorize in the logger configuration.

**Comment by chwba** [comment]
Of course I have already tried that with the same result
